manifests/y/YutakaSawada/MultiPar/1.3.2.6/YutakaSawada.MultiPar.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: YutakaSawada.MultiPar
```

```suggestion
PackageIdentifier: YutakaSawada.MultiPar.Beta
```
This release is a beta version.